pkg/autoscaler/config/config.go

```diff
@@ -200,9 +200,12 @@ func validate(lc *Config) (*Config, error) {
 		return nil, fmt.Errorf("stable-window = %v, must be specified with at most second precision", lc.StableWindow)
 	}
 
-	effPW := time.Duration(lc.PanicWindowPercentage / 100 * float64(lc.StableWindow))
-	if effPW < BucketSize || effPW > lc.StableWindow {
-		return nil, fmt.Errorf("panic-window-percentage = %v, must be in [%v, 100] interval", lc.PanicWindowPercentage, 100*float64(BucketSize)/float64(lc.StableWindow))
+	// We ensure BucketSize in the `MakeMetric`, so just ensure percentage is in the correct region.
+	if lc.PanicWindowPercentage < autoscaling.PanicWindowPercentageMin ||
+		lc.PanicWindowPercentage > autoscaling.PanicWindowPercentageMax {
```

Comment:
What's the relationships between `BucketSize` and `PanicWindowPercentageMin`, `StableWindow` and `PanicWindowPercentageMax`? Are  `PanicWindowPercentageMin` and  `PanicWindowPercentageMax` recently introduced?

Reply:
There's no direct relation. But we cannot have a window smaller than bucketsize.
No. They have been there as long as I remember.